diff --git a/dist/chunk-QLC3UBDY.cjs b/dist/chunk-QLC3UBDY.cjs
index cabe99810b79295e38c6f58a603c0bd41cbc9eb6..91f5f6dd41ee985257d4dd64495ad99f4a1c6dd6 100644
--- a/dist/chunk-QLC3UBDY.cjs
+++ b/dist/chunk-QLC3UBDY.cjs
@@ -20852,7 +20852,7 @@ ${errorMessages}`,
           instructions: v4.z.string().nullish().describe(
             "Additional instructions to append to the agent instructions. Only provide if you have specific guidance beyond what the agent already knows. Leave empty in most cases."
           ),
-          maxSteps: v4.z.number().min(3).nullish().describe("Maximum number of execution steps for the sub-agent")
+          maxSteps: v4.z.coerce.number().min(3).nullish().describe("Maximum number of execution steps for the sub-agent")
           // using minimum of 3 to ensure if the agent has a tool call, the llm gets executed again after the tool call step, using the tool call result
           // to return a proper llm response
         });
diff --git a/dist/chunk-IYSW4VN7.js b/dist/chunk-IYSW4VN7.js
index 69fd21703d9fd2d3fb54d02b5a04decd1fba8e9f..fd5152202152c7f330de60c92b50bb61d44afedf 100644
--- a/dist/chunk-IYSW4VN7.js
+++ b/dist/chunk-IYSW4VN7.js
@@ -20827,7 +20827,7 @@ ${errorMessages}`,
           instructions: z.string().nullish().describe(
             "Additional instructions to append to the agent instructions. Only provide if you have specific guidance beyond what the agent already knows. Leave empty in most cases."
           ),
-          maxSteps: z.number().min(3).nullish().describe("Maximum number of execution steps for the sub-agent")
+          maxSteps: z.coerce.number().min(3).nullish().describe("Maximum number of execution steps for the sub-agent")
           // using minimum of 3 to ensure if the agent has a tool call, the llm gets executed again after the tool call step, using the tool call result
           // to return a proper llm response
         });
